modules/prometheus-service-monitor.adoc

```diff
@@ -8,6 +8,11 @@
 [role="_abstract"]
 If you are using the Prometheus Operator, you can use a service monitor to scrape the metrics from {rh-rhacs-first}.
 
+[IMPORTANT]
+====
+Prometheus Operator monitoring is enabled by default for {osp} 4. No additional configuration is needed.
```

Comment:
```suggestion
{osp} monitoring is enabled by default for {osp} 4. No additional configuration is needed.
```

What are you trying to say here? This section is specifically for people using a custom Prometheus operator, where they do need additional configuration. I think what you're trying to say is that they can also just use OpenShift monitoring instead, but that is not really clear from the sentence.

Reply:
OK, how we can improve this sentence?